docs/source/tutorial/08-add-a-details-view.mdx

```diff
@@ -118,19 +118,22 @@ fun LaunchDetails(launchId: String) {
 }
 ```
 
-## Handle protocol errors
+## Handle errors
 
-As you execute your query, two types of errors can happen:
+As you execute your query, different types of errors can happen:
 
-* Protocol errors: connectivity issues, HTTP errors, or JSON parsing errors. In this case, `response.exception` will contain an `ApolloException`, and `response.data` will be null.
-* Application errors. In this case, `response.errors` will contain the application errors and `response.data` might be `null`.
+* Fetch errors: connectivity issues, HTTP errors, JSON parsing errors, etc... In this case, `response.exception` contains an `ApolloException`. Both `response.data` and `response.errors` is null.
```

Comment:
So should it be "transport errors" then? :)

Reply:
We have "fetch errors" in a lot of places already and I don't want to go back and forth about this. "fetch" is working OK for us (it's not working on web because `fetch` is a different thing there) so I'd rather wait to see if we settle on "transport errors" and do the search and replace all at once.

Reply:
Makes sense 👍